packages/cinterop/src/darwinCommon/kotlin/io/realm/interop/RealmInterop.kt

```diff
@@ -257,6 +257,10 @@ actual object RealmInterop {
         checkedBooleanResult(realm_wrapper.realm_commit(realm.cptr()))
     }
 
+    actual fun realm_rollback(realm: NativePointer) {
+        checkedBooleanResult(realm_wrapper.realm_rollback(realm.cptr()))
```

Comment:
I already changed it from something like `throwOnError` to this. It is implemented to check the C-API for errors if the value is `false` and if no error is registered just return false. I guess we wouldn't have to propagate the `true` return value when the alternative is that we will throw an exception. Think it will make most sense to adjust as part of #70.